labkey-client-api/README.md

```diff
@@ -0,0 +1,69 @@
+# The LabKey Remote API Library for Java
+
+Welcome to the LabKey Remote API library for Java. This library allows Java 
+developers to access the data and services exposed from a LabKey Server installation.
+
+For more information on the library, 
+see our [JavaAPI](https://www.labkey.org/Documentation/wiki-page.view?name=javaAPI) page, 
+which includes a link to the [JavaDoc](https://www.labkey.org/download/clientapi_docs/java-api/)
+where you can get details about the classes and programming interfaces. 
+
+If you have any questions about this library or the LabKey Server, 
+see our [Community Support Forums](https://www.labkey.org/home/Support/LabKey%20Support%20Forum/project-begin.view?).
+
+This library is licensed under the [Apache 2.0 open-source license](http://www.apache.org/licenses/LICENSE-2.0).
+
+See the [change log](CHANGELOG.md) for information on the release versions and their
+compatibility with LabKey Server versions.
+
+## Development
+
+### Dependency Declaration
+To declare a dependency on this jar file, you can use the following in Gradle
+
+```compile(group: 'org.labkey.api', name: 'labkey-client-api', version: '1.2.0')```
+
+If using the LabKey Gradle plugins and building a LabKey module, it is best to 
+use this utility method instead to facilitate testing of any local changes to
+this jar file with that module
+```
+BuildUtils.addLabKeyDependency(project: project, config: "remoteApi", depProjectPath: BuildUtils.getRemoteApiProjectPath(gradle), depVersion: project.labkeyClientApiVersion)
+```
+
+### Making Changes
+
+Though this repository contains its own Gradle version and settings file and can be built
+independently of the rest of LabKey server, you can also include it in the larger LabKey 
+Server Gradle project if you need to make changes to the code and test them locally without
+having to publish.  
+
+- Create a feature branch of this repository (`fb_myNewFeature`)
```

Comment:
nit: These first two steps seem out of order to me. 1) Clone the repo, 2) Make a feature branch.

Reply:
Depends on how you usually make your feature branches, I guess, but sure.